scripts/src/metrics.ts

```diff
@@ -45,6 +45,10 @@ export function reportServerError(method: string, path: string) {
 	statsd.increment("server_error", 1, undefined, { method, path });
 }
 
+export function reportProcessAbort() {
```

Comment:
maybe attach a reason instead of system: exit? i.e. reason: "multiple 5xx errors in short period"

Reply:
the reason can be a parameter